includes/install-scripts/fresh-install.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+echo "BlueSpice installation is started. This process can take up to 10 minutes."
+rm -Rf /data/www
+sleep 2
+echo "."
+source $DOWNLOAD_WIKI_SCRIPT
+# setup mysql
+rm -Rf /data/mysql >>/dev/logs 2>&1
+rm -Rf /var/lib/mysql >>/dev/logs 2>&1
+/usr/bin/mysql_install_db --force --datadir=/data/mysql >>/dev/logs 2>&1
+ln -s /data/mysql /var/lib/mysql >>/dev/logs 2>&1
+source $START_SERVICES_SCRIPT
+
+/usr/bin/mysql -u root -e "CREATE DATABASE bluespice"
+/usr/bin/mysql -u root -e "CREATE USER 'bluespice'@'localhost' IDENTIFIED BY \"$rndpass\""
+/usr/bin/mysql -u root -e "GRANT ALL PRIVILEGES ON bluespice.* to 'bluespice'@'localhost'"
+/usr/bin/mysql -u root -e "FLUSH PRIVILEGES"
+sleep 5
+if [ -z $BS_LANG ]; then
+    BS_LANG="en"
+fi
+if [ -z $BS_URL ]; then
+    BS_URL="http://localhost"
+fi
+if [ -z $BS_USER ]; then
+    BS_USER="WikiSysop"
+fi
+if [ -z $BS_PASSWORD ]; then
+    BS_PASSWORD="PleaseChangeMe"
+fi
+if [ -f "/data/cert/ssl.cert" ] && [ -f "/data/cert/ssl.key" ]; then
+    sed -i "s/{CERTFILE}/\/data\/cert\/ssl.cert/g" /etc/nginx/sites-available/bluespice-ssl.conf
```

Comment:
If HTTPS is used, will Parsoid work? I am assuming that this will most likely be a self-signed certificate, which will make the cURL calls to Parsoid fail usually.

Reply:
Let me check

Reply:
this part is not changed from last time, also its true that parsoid is failing, but if we could make another different issue for this particular thing that would be nice 😟